lexer/src/main/kotlin/tools/samt/lexer/Lexer.kt

```diff
@@ -287,7 +287,7 @@ class Lexer private constructor(
         currentPosition.copy(
                 charIndex = currentPosition.charIndex + 1,
                 row = currentPosition.row + 1,
-                col = -1,
+                col = 0,
```

Comment:
This might cause a problem given multiline input, the first line tokens are 0-indexed but with this change the \
See the failing unit test for a case which breaks with just this change:
https://github.com/samtkit/core/blob/e03554a2a06c8c27c52f280b286755249a2f0e86/lexer/src/test/kotlin/tools/samt/lexer/LexerTest.kt#L264-L283